units/src/fee_rate/mod.rs

```diff
@@ -260,41 +260,41 @@ mod tests {
 
     #[test]
     fn checked_add() {
-        let f = FeeRate::from_sat_per_kwu(1).checked_add(2).unwrap();
-        assert_eq!(FeeRate::from_sat_per_kwu(3), f);
+        let fee_rate = FeeRate::from_sat_per_kwu(1).checked_add(2).unwrap();
+        assert_eq!(fee_rate, FeeRate::from_sat_per_kwu(3));
 
-        let f = FeeRate::from_sat_per_kwu(u64::MAX).checked_add(1);
-        assert!(f.is_none());
+        let fee_rate = FeeRate::from_sat_per_kwu(u64::MAX).checked_add(1);
+        assert!(fee_rate.is_none());
     }
 
     #[test]
     fn checked_sub() {
-        let f = FeeRate::from_sat_per_kwu(2).checked_sub(1).unwrap();
-        assert_eq!(FeeRate::from_sat_per_kwu(1), f);
+        let fee_rate = FeeRate::from_sat_per_kwu(2).checked_sub(1).unwrap();
+        assert_eq!(f, FeeRate::from_sat_per_kwu(1));
```

Comment:
```suggestion
        assert_eq!(fee_rate, FeeRate::from_sat_per_kwu(1));
```
I think that is all the error was about.

Reply:
Thanks!